HW1/P2/P2b.py

```diff
@@ -1,3 +1,35 @@
+# Initialize SC context
+import findspark
+findspark.init()
+import pyspark
+sc = pyspark.SparkContext(appName="Spark 1")
 from P2 import *
 
 # Your code here
+if __name__ == '__main__':
+  # Set up Initial RDDs with 100 partitions
+  rdd = sc.parallelize(range(2000),10)
+  rdd1 = sc.parallelize(range(2000),10)
+  rdd2 = rdd1.cartesian(rdd)
+
+  # RDD repartitioned using partitionBy
+  rdd2_new = rdd2.partitionBy(100)
+
```

Comment:
I think you could have even done a better distribution if you had assigned it randomly rather than by the order of partitioning that partitionBy gives you.